tests/system/v1/conftest.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""System tests for reading rows from tables."""
+
+import os
+import uuid
+
+import pytest
+
+from google.cloud import bigquery_storage_v1
+
+_TABLE_FORMAT = "projects/{}/datasets/{}/tables/{}"
+
+_ASSETS_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)), "../assets")
+
+
+@pytest.fixture(scope="session")
+def project_id():
+    return os.environ["PROJECT_ID"]
+
+
+@pytest.fixture(scope="session")
+def credentials():
+    from google.oauth2 import service_account
+
+    # NOTE: the test config in noxfile checks that the env variable is indeed set
+    filename = os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+    return service_account.Credentials.from_service_account_file(filename)
+
+
+@pytest.fixture(scope="session")
+def client(credentials):
+    return bigquery_storage_v1.BigQueryReadClient(credentials=credentials)
+
+
+@pytest.fixture()
+def table_reference():
+    return _TABLE_FORMAT.format("bigquery-public-data", "usa_names", "usa_1910_2013")
+
+
+@pytest.fixture()
+def small_table_reference():
+    return _TABLE_FORMAT.format(
+        "bigquery-public-data", "utility_us", "country_code_iso"
+    )
+
+
+@pytest.fixture(scope="session")
+def local_shakespeare_table_reference(project_id):
+    return _TABLE_FORMAT.format(project_id, "public_samples_copy", "shakespeare")
```

Comment:
@shollyman I'm curious why the local copy of `bigquery-public-data.samples.shakespeare`? I'm adding a script to create this table in https://github.com/googleapis/python-bigquery-storage/pull/74